src/_includes/layouts/home.njk

```diff
@@ -1,11 +1,12 @@
 {% extends "layouts/base.njk" %}
 
-{% block content %}
-    <article class="h-entry [ post ] [ flow wrapper ]">
-        <h1 class="p-title">{{ title }}</h1>
+{% block homecontent %}
+  <div class="home-wrapper">
+    <main tabindex="-1" id="main-home">
```

Comment:
This might be my lacking knowledge here, but does `tabindex="-1"` take all of the content of the site out of the tab order?

Reply:
Child content should remain in tab order. It makes the main content container focusable but not within the default tab focus order. I think this is common practice with "Skip to content" links to ensure the next focus, after skipping to the main content, is the next focusable element within, instead of the next focusable element after the "Skip to content" link.

Reply:
Thank you for the insight!